src/Sylius/Behat/Context/Ui/Admin/ManagingShipmentsContext.php

```diff
@@ -29,9 +31,13 @@ final class ManagingShipmentsContext implements Context
     /** @var NotificationCheckerInterface */
     private $notificationChecker;
 
-    public function __construct(IndexPageInterface $indexPage, NotificationCheckerInterface $notificationChecker)
+    /** @var ShowPageInterface */
+    private $orderShowPage;
```

Comment:
Change the order of services (also parameters of constructor) so that the `NotificationCheckerInterface` is the last one